apps/src/applab/commands.js

```diff
@@ -1776,3 +1777,4 @@ function stopLoadingSpinnerFor(elementId) {
 
 // Include playSound, stopSound, etc.
 Object.assign(applabCommands, audioCommands);
+Object.assign(applabCommands, makerCommands);
```

Comment:
TODO (future PR): Other maker 'commands' are defined right here in applab/commands.js - we should move them out to maker/commands.js.